otel.yml

```diff
@@ -24,6 +25,9 @@ extensions:
   health_check:
     endpoint: "localhost:13133"
     path: "/health/status"
+  file_storage/filelogreceiver:
+    create_directory: true
+    directory: ${env:STORAGE_DIR}
 
 service:
   extensions: [health_check, memory_limiter]
```

Comment:
The `file_storage/filelogreceiver` extension must also be added to `service::extensions` list, right?